portfolio/src/main/webapp/script.js

```diff
@@ -12,23 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// A list of activities that could appear when clicking activities button.
-const activities = ['Coding 💻', 'Travelling 🗼', 'Brewing coffee ☕', 'Going on a nature walk 🌲'];
+// This is a list of passions to iterate through on the homepage
+const passions = ["Front-End Development?", "Machine Learning?", "Software Design?", "Technology Consulting?" ,"Data Science?", "Algorithms?", "User Experience?"];
+const typewriterLetterDelay = 100;
+const typewriterWordDelay = 1000;
```

Comment:
The delay constants need their own documentation. In particular, it's not clear what is the unit of time. Also consider changing the variable name to end in "Ms" or "Milliseconds" so it's really clear.

Reply:
Good Catch - fixed. Thanks!